python/pyarrow/tests/interchange/test_conversion.py

```diff
@@ -16,11 +16,16 @@
 # under the License.
 
 from datetime import datetime as dt
-import numpy as np
 import pyarrow as pa
 from pyarrow.vendored.version import Version
 import pytest
 
+try:
+    import numpy as np
+except ImportError:
+    pytest.skip(reason="Failures on test collection due to numpy NOT enabled",
+                allow_module_level=True)
```

Comment:
Perhaps open a GH issue to fix this, and reference it here?

Reply:
Looking at the file here, it seems the actual parametrization using numpy are just float32/float64 dtypes, and those could easily be converted to strings